packages/esm-patient-programs-app/src/routes.json

```diff
@@ -27,6 +27,10 @@
         "path": "Programs"
       },
       "order": 10
+    },
+    {
```

Comment:
This should go in the `modals` array. Please read the [Modals docs](https://o3-dev.docs.openmrs.org/docs/modal-system).